src/ape/cli/commands.py

```diff
@@ -6,6 +6,16 @@
 from ape import networks
 
 
+def check_parents_for_interactive(ctx: Context) -> bool:
```

Comment:
**praise**: The name of the method cracks me up!

Reply:
🤣